stdlib/stdlib-external/runtime/string.ts

```diff
@@ -546,7 +555,7 @@ function grainHeapValueToString(ptr: u32, extraIndents: u32): u32 {
   }
 }
 
-function grainToStringHelp(grainValue: u32, extraIndents: u32): u32 {
+function grainToStringHelp(grainValue: u32, extraIndents: u32, top: bool): u32 {
```

Comment:
Maybe a better name than `top` here? I know for a fact in 6 months I won't have any idea what this means